### PR TITLE
Ft ui ux rework 147181013

### PR DESCRIPTION
**Pull request by Alweezy**
This alerts the user on the availability of a feature to collaboratively create and manage tasks.

### CODE REVIEW

**Comment by Alweezy** [comment]
Follow [this link](https://ui-ux-fix.herokuapp.com/) to heroku deploy.

**Comment by NdagiStanley** [comment]
@Alweezy let's fix the merge conflicts.

**Comment by Alweezy** [comment]
@NdagiStanley conflicts resolved.

File: hc/wsgi.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 from whitenoise.django import DjangoWhiteNoise
 
 os.environ.setdefault("DJANGO_SETTINGS_MODULE", "hc.settings")
+from whitenoise.django import DjangoWhiteNoise
```
Comment (NdagiStanley) [inline hc/wsgi.py]: Please note that line number `13` and `16` are the same.

File: requirements.txt [hunk 2]
```diff
@@ -10,3 +10,10 @@ premailer==2.9.6
 psycopg2==2.6.1
 requests==2.9.1
 whitenoise==3.3.0
+boto==2.47.0
+cssselect==1.0.1
+cssutils==1.0.2
+lxml==3.8.0
+rcssmin==1.0.6
+rjsmin==1.0.12
```
Comment (NdagiStanley) [inline requirements.txt]: Note that these are dependency libraries. Please confirm this.

**Review by NdagiStanley — CHANGES_REQUESTED** [review]
Looks nice. I'll have a look at the deployment. In the meantime address the highlighted comments.
Feel free to push back.

File: requirements.txt (hunk 2 above)
Comment (Alweezy) [reply requirements.txt]: That's true, let me solve that.

**Comment by Alweezy** [comment]
Please check out, I have resolved the earlier communicated issues

File: .gitignore [hunk 3]
```diff
@@ -6,4 +6,5 @@ hc/local_settings.py
 static-collected
 
 .DS_Store
-static/.DS_Store
\ No newline at end of file
+static/.DS_Store
+.idea
```
Comment (NdagiStanley) [inline .gitignore]: Good thing you added the `.idea` folder here.

**Review by NdagiStanley — APPROVED** [review]

**Pull request merged**